examples/cookbook/agents/localized_agent.py

```diff
@@ -124,7 +128,9 @@ async def _step(self, question: str) -> openai.OpenAIDynamicConfig:
             ]
         }
 
-    async def _get_response(self, question: str):
+    async def _get_response(
+        self, question: str
+    ) -> Coroutine[Any, Any, Any | None] | None:
```

Comment:
doesn't async automatically type it as a coroutine? this makes it seem like the result of awaiting this function would be another function to await? why not just `None` in line with the other `_get_response`?